roofit/roofitcore/test/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ ROOT_ADD_GTEST(testRooWrapperPdf testRooWrapperPdf.cxx LIBRARIES Gpad RooFitCore
 ROOT_ADD_GTEST(testGenericPdf testGenericPdf.cxx LIBRARIES RooFitCore)
 ROOT_ADD_GTEST(testRooAbsPdf testRooAbsPdf.cxx LIBRARIES RooFitCore)
 ROOT_ADD_GTEST(testRooDataSet testRooDataSet.cxx LIBRARIES Tree RooFitCore)
-ROOT_ADD_GTEST(testRooFormula testRooFormula.cxx LIBRARIES RooFitCore)
+ROOT_ADD_GTEST(testRooFormula testRooFormula.cxx LIBRARIES RooFitCore NO_FAILREGEX) #Override failregex, since we are actually testing failures.
```

Comment:
Do you mean testing 'expected failures'? Maybe WILLFAIL can help?

Reply:
No, I meant that the test is testing that things are in a sane state even after errors have been encountered. That's done with gtest `EXPECT`s. However, the test will also print error messages, but these shouldn't lead to failures.
With `WILLFAIL`, the gtest part wouldn't work properly. With `NO_FAILREGEX`, output simply won't be checked at all.

Reply:
A solution is to capture the TError messages, and have the output checked with the FAILREGEX, but this requires the capturing to work.

Reply:
(indeed the consensus at the ROOT meeting was that this PR depends on #5297 )

Reply:
Okay, got it. Thanks!

Reply:
Ah, sorry. I wanted to have tagged the other one ...